crates/blockifier/src/execution/contract_class.rs

```diff
@@ -648,9 +624,8 @@ impl NativeContractClassV1 {
 pub struct NativeContractClassV1Inner {
     pub executor: AotNativeExecutor,
     entry_points_by_type: NativeContractEntryPoints,
-    // Storing the raw sierra program and entry points to be able to fallback to the vm
-    sierra_program_raw: Vec<BigUintAsHex>,
-    fallback_entry_points_by_type: SierraContractEntryPoints,
+    // Used for PartialEq
+    sierra_program_hash: BlockHash,
```

Comment:
Is using `BlockHash` here ok? Maybe it would be better to using something like `StarkHash`?

Reply:
I think BlockHash is good here. StarkHash causes some issues because it's a trait or I think somewhere else it's just defined as a Felt.